### DISCUSSION TITLE
What are your extensions on Patient, Practitioner, PractitionerRole, Organization, AllergyIntolerance?

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#44] [Consensus Discussions for R1]
_We want the community to have the necessary support for the concepts you need._

Comment to identify the extensions you use for the following resources:
- Patient
- Practitioner
- PractitionerRole
- Organization
- AllergyIntolerance

What IG-specific extensions are you using for each resource that your organisation has needed to define?
What AU extensions (see AU Base) are you are using for each resource?
What international extensions are you using for each resource?

**Comment by davidmckillop-ADHA** [comment]
The following extensions are used in the Australian Digital Health Agency profiles:

Resource Profile | Extension | Publisher 
-- | -- | -- |
Patient | [birthPlace](http://hl7.org/fhir/R4/extension-patient-birthplace.html) |  HL7
-- | [interpreterRequired](http://hl7.org/fhir/R4/extension-patient-interpreterrequired.html) | HL7
-- |  [birthTime](http://hl7.org/fhir/R4/extension-patient-birthtime.html) | HL7
-- | [Australian Indigenous Status](https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-indigenous-status.html) | HL7 AU
-- | [Date Accuracy Indicator](https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-date-accuracy-indicator.html) | HL7 AU
-- | [Date of Arrival in Australia](https://build.fhir.org/ig/hl7au/au-fhir-base//StructureDefinition-date-of-arrival.html) | HL7 AU
 Practitioner | None | 
 PractitionerRole | None | 
 Organization | None |
 AllergyIntollerance | None |

**Comment by reubendaniels1** [comment]
Please note the following responses from Queensland Health

**Extensions used for Patient**

Name | Publication | Publisher
-----|-------------|----------
birthPlace|FHIR standard|HL7 International
birthTime|FHIR standard|HL7 International
individual-genderIdentity|FHIR standard|HL7 International
individual-pronouns|FHIR standard|HL7 International
individual-recordedSexOrGender|FHIR standard|HL7 International
interpreterRequired|FHIR standard|HL7 International
patient-sexParameterForClinicalUse|FHIR standard|HL7 International
auTime|AU Base IG (CI / 4.1.1)|HL7 Australia
dateAccuracyIndicator|AU Base IG (CI / 4.1.1)|HL7 Australia
dateOfArrival|AU Base IG (CI / 4.1.1)|HL7 Australia
deceasedBoolean|AU Base IG (CI / 4.1.1)|HL7 Australia
deceasedDateTime|AU Base IG (CI / 4.1.1)|HL7 Australia
ethnicity|AU Base IG (CI / 4.1.1)|HL7 Australia
ihiStatus|AU Base IG (CI / 4.1.1)|HL7 Australia
ihiRecordStatus|AU Base IG (CI / 4.1.1)|HL7 Australia
ihiVerifiedDate|AU Base IG (CI / 4.1.1)|HL7 Australia
indigenousStatus|AU Base IG (CI / 4.1.1)|HL7 Australia
noFixedAddress|AU Base IG (CI / 4.1.1)|HL7 Australia

**Locally required extensions for Patient**

Name | Description | Datatype
-----|-------------|---------
Australian South Sea Islander|Whether a person identifies as being of Australian South Sea Islander ancestry. |Code. Values: _Yes_, _No_, _Not stated/unknown_
Enterprise Unique Identifier (EUID)|State-wide patient identifier held and assigned by the Queensland Master Patient Index (QMPI) – Queensland Health’s enterprise master patient index solution|Identifier
Source of death notification|Describes the source of information relating to a person's death.|Code. Values: _Official death certificate or death register_, _Healthcare provider_, _Relative_, _Approved agency_, _Re-instated_, _Other_, _Unknown_
Variations of sex characteristics|An indicator whether an individual has innate genetic, hormonal or physical sex characteristics that do not conform to medical norms for female or male bodies. It refers to a wide spectrum of variations to genitals, hormones, chromosomes and/or reproductive organs.|Code. Values: _Yes_, _No_, _Do not know_, _Prefer not to answer_  

**Extensions used for Practitioner**

Name | Publication | Publisher
-----|-------------|----------
individual-genderIdentity|FHIR standard|HL7 International
qualification.ahpraProfession-details|AU Base IG (CI / 4.1.1)|HL7 Australia
qualification.ahpraRegistration-details|AU Base IG (CI / 4.1.1)|HL7 Australia

**Locally required extensions for Practitioner**

Name | Description | Datatype
-----|-------------|---------
Registration board|The name of the organisation that manages the registration of the individual service provider. Apart from AHPRA, this may include organisations like the Dietician's Association of Australia and Speech Pathology Australia.|Text

**Extensions used for PractitionerRole**

Name | Publication | Publisher
-----|-------------|----------
timeZone|AU Base IG (CI / 4.1.1)|HL7 Australia

**Locally required extensions for Organization**
Name | Description | Datatype
-----|-------------|---------
Name usage|Enables differentiation between recorded names for an organisation or service location by their intended application.|Code. Values: _Unit/section/division name_, _Service location name_, _Business name_, _Locally used name_, _Abbreviated name_, _Organisation name_, _Other_, _Unknown_
